client/state/preferences/selectors.js

```diff
@@ -36,6 +36,26 @@ export function getPreference( state, key ) {
 	);
 }
 
+export function getMediaScalePreference( state, key, isMobile ) {
+	const SCALE_TOUCH_GRID = 0.323;
```

Comment:
The `SCALE_TOUCH_GRID` constant is not duplicated at two places. Let's move it to the `lib/media/constants` module, right next to the `SCALE_CHOICES` array, which is very closely related.